src/flash-source-buffer.js

```diff
@@ -367,12 +364,18 @@ export default class FlashSourceBuffer extends videojs.EventTarget {
     targetPts *= 1e3;
     targetPts += this.basePtsOffset_;
 
-    // skip tags with a presentation time less than the seek target/end of buffer
-    for (let i = 0; i < audioTags.length; i++) {
-      if (audioTags[i].pts >= targetPts) {
-        filteredAudioTags.push(audioTags[i]);
+    // gets the index of the key frame based on whether the tag is a metadata tag or not
+    const getKeyFrameIndex = (tag, index) => {
```

Comment:
These functions should just be defined once at the top of the module. They don't appear to rely on closure variables and the overhead of recreating them each time `convertTagsToData_` is executed is wasteful - especially in IE where there is already a poorly performing VM.